defaults/main.yml

```diff
@@ -30,6 +30,11 @@ sentry_dsn:
 google_analytics_tracking_id:
 extra_environment_variables: {}
 
+# Traefik settings
+traefik:
+  extra_config_path: /etc/hub/traefik
+  docker_extra_config_path: /custom_config
```

Comment:
two last points: 

a) this deserves documentation in the readme.md
b) there is this slight ugly thing in yaml and the way variables are overridden: If the calling playbook / ansible code does not contain any variable called `traefik`, the one defined here will come into play. If, however, the calling code contains a variable `traefik` and it is not a dictionary (but just a simple string) or it is a dictionary but does not contain both options (`extra_config_path`, `docker_extra_config_path`), these options will be gone and ` tasks/reverse_proxy.yml ` will fail. That is why for the most (nested) parameters in here, we also offer a flat version to easily override only some values without having to specify the full dictionary, see e.g. lines 69ff which references the flat variables from line 12ff

Reply:
a) addressed in https://github.com/Innoactive/ansible-innoactive-hub-role/pull/100/commits/62d607a554c23edf41a8fa157a54f4e36e22ccb6
b) addressed in https://github.com/Innoactive/ansible-innoactive-hub-role/pull/100/commits/9dc7b3bbbed3207ed49019a82517f86d3b70f7d9